examples/offloader.js

```diff
@@ -31,53 +33,54 @@ if (fs.existsSync('polldata.json')) {
 
 client.logOn(logOnOptions);
 
-client.on('loggedOn', function() {
-	console.log("Logged into Steam");
+client.on('loggedOn', function () {
+	console.log('Logged into Steam');
 });
 
-client.on('webSession', function(sessionID, cookies) {
-	manager.setCookies(cookies, function(err) {
```

Comment:
That's not what I was referring to. You shouldn't need `err` in more than the initial scope where it's available, and nearly always at the very top of that callback. Shadowing it doesn't matter in the vast majority of cases is fine since all you're losing reference to is `null`.